lib/peep.rb

```diff
@@ -0,0 +1,13 @@
+require 'pg'
+class Peep
+  def self.all
+    connection = PG.connect(dbname: 'Peeps')
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peep| peep['peep'] }
```

Comment:
Whilst this is relatively easy to understand as someone who has completed to task too, could one of the peeps on line 6 be renamed so it is clearer what these represent? Good use of result.map though to iterate through the list of all peeps!